packages/transition-backend/src/services/accessMapLocation/AccessMapLocationProvider.ts

```diff
@@ -29,19 +29,22 @@ export interface accessMapLocationOptions {
     debug?: boolean;
 }
 
+const DEFAULT_PROJECTION_SRID = 4326;
```

Comment:
Now that I see this, it should be a preference itself...

Reply:
True, since a user will probably always use the same projection.

Reply:
This is actually a server configuration, as the FIXME I'll add explains